Dockerfiles/midstream/run_tests.py

```diff
@@ -68,13 +68,15 @@ def test_for_extract_and_validate_bundle_image(self):
             exit_code = 103
             return exit_code
         result = self.run_extract_operator_bundle()
+        print(result.stdout)
         if (result.returncode != 0):
             print("Ansible playbook extract-operator-bundle.yml failed with result code: %s , see the file .errormessage for more info." % result.returncode)
             with open(".errormessage", "w") as error_msg:
                 print("Result code: " + str(result.returncode), "Error message: " + str(result.stderr), file=error_msg)
             exit_code = 50
             return exit_code
         result = self.run_validate_operator_bundle()
+        print(result.stdout)
```

Comment:
Will midstream capture stdout somewhere? If no than we need to log it also to some file
Cause we will have to be able debug anything just by reading logs.

Reply:
@lslebodn Midstream has a jenkins console where they can stdout to.